R/colour-text.r

```diff
@@ -1,8 +1,9 @@
-
+# fix the check NOTE:Namespace in Imports field not imported from: ‘crayon’
```

Comment:
I would say they are accessed, not imported. i.e the symbols are not copied in the imports env, aren't they ?